src/controllers/translateController.ts

```diff
@@ -40,31 +41,41 @@ const translateController = async ({
     languages,
     basePath,
     translationEngines: settingsTranslationEngines,
+    sort,
   } = await importJsonFile<Settings>(settingsFilePath);
 
+  const sortedLanguages = sortElements(
+    languages.map((lang) => lang.name),
+    sort as "A-Z" | "Z-A" | "none"
```

Comment:
this "as" should not be needed as it is already declared on src/types/settings.d.ts